site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -287,15 +295,19 @@ from fusionauth.fusionauth_client import FusionAuthClient
 ```
 
 Next, we'll need to add our API Key and URL for the FusionAuth server.
-Yours will be different. If you don't remember your API Key, navigate to **Settings-->API Keys**.
+Yours will be different. If you don't remember your API Key, navigate to "**Settings**" and then to "API Keys".
 
-Your API Key should be in the "ID" field. If you're missing an API Key, you'll need to create one to continue. Once you have your API key, replace `YOUR-OWN-API-KEY` with your own in the code below:
+Your API Key should be in the "Id" field. If you're missing an API Key, you'll need to create one to continue. Once you have your API key, replace `YOUR-OWN-API-KEY` with your own in the code below:
 
 ```python
 #  You must supply your API key and URL here
 client = FusionAuthClient('YOUR-OWN-API-KEY', 'http://localhost:9011')
 ```
 
+{% include _callout-tip.liquid content=
+"The base url provided to the client constructor should not contain a trailing ' / ' "
```

Comment:
Love the call out, please add a period at the end.

Reply:
fixed in latest commit